internal/osbuild/greenboot_stage.go

```diff
@@ -0,0 +1,14 @@
+package osbuild
+
+type GreenbootConfig struct {
+	MonitorServices []string `json:"monitor_services,omitempty"`
+}
```

Comment:
I missed this originally but this doesn't match the osbuild stage schema.
This will produce stage as

```json
{
  "type": "org.osbuild.greenboot",
  "options": {
    "monitor_services": [
      "sshd",
      "NetworkManager"
    ]
  }
}
```

when it should be

```json

{
  "type": "org.osbuild.greenboot",
  "options": {
    "config": {
      "monitor_services": [
        "sshd",
        "NetworkManager"
      ]
    }
  }
}

```

In other words, we need one more struct for the options which is

```
type GreenbootOptions struct {
  Config GreenbootConfig `json:"config"`
}
```
and then that should be the one that implements `isStageOptions()`.

Reply:
fixing it